draft-ietf-tls-tls13.md

```diff
@@ -3896,9 +3898,9 @@ TLS records are typed, which allows multiple higher-level protocols to
 be multiplexed over the same record layer. This document specifies
 four content types: handshake, application data, alert, and
 change_cipher_spec, with the latter being used only for
-compatibility purposes (see {{middlebox}}).
+compatibility purposes (see {{intermediary}}).
 
-An implementation may receive an unencrypted record of type
+{#dummy_ccs}An implementation may receive an unencrypted record of type
```

Comment:
There is another unsupported inline anchor here. It seems also unhappy about the use of `client_hello`, maybe try `client-hello`?

Reply:
Thanks, I fixed this. And I moved a sentence from the 2nd to 3rd paragraph.